### PR TITLE
Remove redis

### PR DESCRIPTION
**Pull request by deathbearbrown**
:D finally.

This fixes all the tasks of sxi-174 and sxi-173

### CODE REVIEW

File: server/database.js [hunk 1]
```diff
@@ -74,6 +74,22 @@ Database.prototype.runScriptFile = function(fileName) {
...
+Database.prototype.insertApplicationConfig = function(data) {
+  var columns = ['accountId', 'organization', 'mqttUser', 'device', 'endUser'];
+  return this._insertQuery('application_config', columns, data);
+};
+
+Database.prototype.selectApplicationConfig = function() {
+  return this._query('SELECT * FROM application_config');
+};
+
+Database.prototype.selectApplicationConfig = function(accountid) {
```
Comment (ksiegel) [inline server/database.js]: I think I got double vision!

File: server/index.js [hunk 2]
```diff
@@ -66,6 +48,37 @@ parsers.use(bodyParser.json());
...
+    env: envData,
+    files: ['meta.json', 'account.json', 'virtualdevice.json'],
+  });
+
+  // Allow config to be referenced by name inside of views.
+  // This creates a new object that has everything in config plus an
+  // additional property "config" that is a circular reference
+  config.view = _.merge({}, config.raw, {config: config.raw});
+});
+
```
Comment (ksiegel) [inline server/index.js]: I think we can get rid of the loading page middleware if we do a `.then(server-start-thing())` after this.. which would be another great reduction in complexity

File: server/database.js (hunk 1 above)
Comment (deathbearbrown) [inline server/database.js]: haha :/  thank you for saving me from myself

File: provision/tables.sql [hunk 3]
```diff
@@ -1,4 +1,4 @@
-DROP TABLE IF EXISTS inventory, firmware; 
+DROP TABLE IF EXISTS inventory, firmware, application_config;
```
Comment (ksiegel) [inline provision/tables.sql]: can you add rules in here? my mistake with my code a while ago

Comment (deathbearbrown) [inline provision/tables.sql]: yep!

**Comment by ksiegel** [comment]
https://github.com/xively/concaria/blob/master/app.json#L16
redis is there too. also reorder that so Xively is on top 

https://github.com/xively/concaria/blob/master/package.json#L79 too

File: server/index.js (hunk 2 above)
Comment (deathbearbrown) [inline server/index.js]: hrmm, can we make that a separate issue?

**Comment by deathbearbrown** [comment]
Removed redis from the package and app json files

File: server/index.js (hunk 2 above)
Comment (ksiegel) [inline server/index.js]: I suppose so, there are some other implications that we'd deal with if there was no loading page due to the db failing

Comment (ksiegel) [inline server/index.js]: actually with that thought we should have a `.catch()` in here in case the db fails

Comment (deathbearbrown) [inline server/index.js]: good call.

**Comment by ksiegel** [comment]
Looks good to merge! :+1:

**Pull request merged**